docs/policy-only.md

```diff
@@ -0,0 +1,166 @@
+1. At Antrea initialization, an ip-tables rule is created in ``mangle table`` that marks IP packets
+with service IP as destination IP and are from gw0. 
+1. At Antrea initialization, an ip-rule is added to select ``antrea_service`` route table as routing
+table if traffic is marked in 1).
+1. At Antrea initialization, a default route entry is added to ``antrea_service`` route table to
+forward all traffic to ``gw0``.
+
+The outcome may be something like this
+```bash
+ip neigh | grep gw0
+169.254.253.1 dev gw0 lladdr 12:34:56:78:9a:bc PERMANENT
+
+ip route show table 300 #tbl_idx=300 is antrea_service
+default via 169.254.253.1 dev gw0 onlink 
+
+ip rule | grep gw0
+300:	from all fwmark 0x800/0x800 iif gw0 lookup 300 
+
+iptables -t mangle  -L ANTREA-MANGLE 
+Chain ANTREA-MANGLE (1 references)
+target     prot opt source               destination         
+MARK       all  --  anywhere             10.0.0.0/16          /* Antrea: mark service traffic */ MARK or 0x800
+MARK       all  --  anywhere            !10.0.0.0/16          /* Antrea: unmark post LB service traffic */ MARK and 0x0
+```
+
+The above configuration allows Pod-to-Service traffic to use ``antrea_service`` route table after
+load balancing, and to be steered back to OVS bridge for Pod NetworkPolicy processing.
+
+#### Conntrack
+Note also that with re-entrance traffic, a service request, after being load balanced and routed
+back to OVS bridge via ``gw0``, has exactly the same 5-tuple as when it re-enters the host network
+for forwarding.
+
+When a service request with same 5-tuples re-enters the host network, it confuses Linux conntrack. 
+The Linux considers the re-entrance IP packet from a new connection flow that uses same source port
+that has been allocated in the DNAT connection. In turn, the re-entrance packet triggers
+another SNAT connection. The overall effect is that the service's DNAT connection is not
+discovered by the service reply, and no Un-DNAT takes place. As a result, the reply is not
+recognized, and therefore dropped by the source Pod.
+   
+Antrea uses the following mechanisms to handle Pod-to-Service traffic re-entrance to the host
+network, and bypasses conntrack in host network.
+1. In OVS bridge, adds flow that marks any re-entrance traffic with a special source MAC.
+1. In OVS bridge, adds flow that causes any re-entrance traffic to bypasses conntrack in OVS zone.
+1. In the host network' ip-tables, adds a rule in ``raw`` table that if matching the special
+source MAC in 1), bypass conntrack in host zone.
+
+#### NetworkPolicy Considerations
+Note that when a traffic flow is re-entrance, the original reply packets do not make into OVS,
+it is un-DNATted in the host network before reaching OVS. This, however, does not have any impact on
+NetworkPolicy enforcement.
+ 
+Antrea enforces NetworkPolicy by allowing or disallowing initial connection packets (e.g. TCP
+ sync) to go through and to establish connection. Once a connection is
```

Comment:
did you mean "SYN" here, or is "TPC sync" some common term I don't know?

Reply:
done.